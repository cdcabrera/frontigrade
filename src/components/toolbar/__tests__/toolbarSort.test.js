import React from 'react';
import { mount } from 'enzyme';
import ToolbarSort from '../toolbarSort';

describe('ToolbarSort Component', () => {
  it('should render a basic component', () => {
    const props = {};

    const component = mount(<ToolbarSort {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
