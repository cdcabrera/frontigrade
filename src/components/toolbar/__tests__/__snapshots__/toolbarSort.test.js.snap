// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarSort Component should render a basic component 1`] = `
<div
  class="form-group"
/>
`;
